modules/acknowledgements/help/acknowledgements.md

```diff
@@ -0,0 +1 @@
+Acknowledgements
```

Comment:
Is there any help text in @llevitis's PR to use here?

Reply:
yes